settings.gradle

```diff
@@ -4,6 +4,10 @@ include ':src:AirshipBindings.Android.Core'
 include ':src:AirshipBindings.Android.FCM'
 include ':src:AirshipBindings.Android.Preference'
 include ':src:AirshipBindings.Android.Accengage'
+include ':src:AirshipBindings.Android.Automation'
+include ':src:AirshipBindings.Android.MessageCenter'
+include ':src:AirshipBindings.Android.Location'
+include ':src:AirshipBindings.Android.HMS'
```

Comment:
This is getting to be a long list. It might be easier if it is sorted alphabetically.